### ISSUE TITLE
 Warpinator icon looks nicer in Windows & Linux vs. Android..

### ISSUE THREAD

**Issue by juanpc2018** [slowscript/warpinator-android#110]
Android Icon looks so weird.
between light purple & unsaturated pink.

Linux:
![image](https://user-images.githubusercontent.com/33388902/178862427-fd0a2430-8b92-4ab0-967a-41748f93ff12.png)

**Comment by juanpc2018** [comment]
![image](https://user-images.githubusercontent.com/33388902/178863797-911bb8ea-ff8d-4eca-8405-5df6d630677e.png)

**Comment by Pentaphon** [comment]
I agree. The Linux icon should be the one used for all platforms since it looks the nicest.

**Comment by Secret-chest** [comment]
Yes, the icon inconsistency is a bit weird. But the app is unofficial. @slowscript  might have to check with Clem if using the original icon is OK.

**Comment by slowscript** [comment]
The icon inconsistency is intentional to make it clear that my clones are not affiliated with the original, as you have correctly pointed out. I also don't think the Linux icon would fit in with either Android or Windows so I'm leaving it as is for now. Android version also supports the Material You theming, so the background color can be different.

**Comment by Pentaphon** [comment]
> The icon inconsistency is intentional to make it clear that my clones are not affiliated with the original, as you have correctly pointed out. I also don't think the Linux icon would fit in with either Android or Windows so I'm leaving it as is for now. Android version also supports the Material You theming, so the background color can be different.

I think what they are trying to say in the nicest way possible is that the icon is a bit ugly and could use a refresh to make it look more like a polished, modern Android icon. You should try seeing if anybody out there would volunteer to make you a custom icon for free in exchange for credit on the app credits, which many up and coming graphics-minded people do online to build a reputation and portfolio of their work.

**Comment by NikunjKhangwal** [comment]
I think it looks fine. Maybe the circles could get a little bigger and one circle can be filled instead of outlined?
